docs/www/configuration.md

```diff
@@ -112,9 +112,12 @@ redpanda:
     name: external
     port: 9093
 
-  # TLS configuration for the Kafka API.
+  # A list of TLS configurations for the Kafka API listeners.
   # Default: null
   kafka_api_tls:
+    # The name of the specific listener this TLS to which this config
+    # will be applied.
+  - name: "external"
```

Comment:
What happens if we actually do this? i.e., don't include "internal"?

I think the example should be consistent in the file, so perhaps 
```yaml
   - name "internal"
     enabled: false
   - name: "external"
     enabled: true
     # other params, with reasonabl config
```

Reply:
Fixed!